packages/terra-heading/src/terra-dev-site/doc/heading/Heading.1.doc.mdx

```diff
@@ -46,10 +47,37 @@ import Heading from 'terra-heading';
 [View component source code](https://github.com/cerner/terra-core/tree/master/packages/terra-heading)
 
 ## Examples
+<p>
```

Comment:
you don't need the p tags

Reply:
updated here https://github.com/cerner/terra-core/pull/2881/commits/41ecf263920b685685d30b748aad9c2760b50c48.